### PR TITLE
Add cheat sheet for Conan 2.0

### PR DESCRIPTION
**Pull request by prince-chrismc**
I will prep the blog PR to announce this as well :) but this can be merged if it looks good!

image

image

### CODE REVIEW

File: images/cheatsheet/tmp.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+temp
```
Comment (memsharded) [inline images/cheatsheet/tmp.txt:1]: Is this file necessary?

File: knowledge/cheatsheet.rst [hunk 2]
```diff
@@ -0,0 +1,16 @@
+***********
+Cheat sheet
+***********
+
+JFrog has created the following visual cheat sheet for basic Conan commands and
+concepts which users can print out and use as a handy reference. It is available
```
Comment (memsharded) [inline knowledge/cheatsheet.rst:6]:
```suggestion
This is a visual cheat sheet for basic Conan commands and
concepts which users can print out and use as a handy reference. It is available
```
Everything in these docs is created by JFrog, this reads a bit weird

**Review by memsharded — COMMENTED** [review]
Looks good, please check minor comments.

**Comment by SSE4** [comment]
usually, it's not good idea to store binary files like .pdf in .git repositories.
@prince-chrismc how was it generated exactly? is it possible to render PDF and PNG via some script during CI build?

also, it would make it easier to edit this cheat sheet if we need.

**Comment by SSE4** [comment]
some comments (can't comment inline):
- typo `eg.` -> `e.g.`
- bad visual line break for `package_inf o()`
- build requires `CMake >= 3.23`, how do I do it for an older version? IIRC tribe set CMake 3.15+ as requirement

File: images/cheatsheet/tmp.txt (hunk 1 above)
Comment (prince-chrismc) [reply images/cheatsheet/tmp.txt]: I needed that to uplaod the files :) deleted

**Review by memsharded — APPROVED** [review]
